lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -141,7 +142,7 @@ class IdentityCreatorController extends BaseController {
     inputNameIdentityController.dispose();
     inputBccIdentityController.dispose();
     scrollController.dispose();
-    richTextWebController.onClose();
+    richTextWebController.dispose();
```

Comment:
Please use `    richTextWebController.onClose();`

Reply:
Why?

Reply:
> Why?

Who are you asking? What are you asking about?